ckanext/stadtzhharvest/harvester.py

```diff
@@ -285,6 +285,13 @@ def _import_package(self, harvest_object):
         package_dict['name'] = munge_title_to_name(package_dict[u'datasetID'])
         context = self._create_new_context()
 
+        # check if dataset must be deleted
+        import_action = package_dict.pop('import_action', 'update')
+        log.debug(import_action)
+        log.debug(package_dict['name'])
+        if import_action == 'delete':
```

Comment:
I think here `harvest_object.current` must be set to `False` (see https://github.com/ckan/ckanext-harvest/blob/3a72337f1e619bf9ea3221037ca86615ec22ae2f/ckanext/harvest/interfaces.py#L124)

Reply:
Thanks, have added that.